libcxx/test/std/utilities/utility/forward/forward_like.verify.cpp

```diff
@@ -39,8 +39,8 @@ void test() {
   (void)std::forward_like<vfp>(i);  // expected-error {{no matching function for call to 'forward_like'}}
   (void)std::forward_like<cvfp>(i); // expected-error {{no matching function for call to 'forward_like'}}
 
-  using fpr  = void()&;
-  using fprr = void()&&;
+  using fpr  = void() &;
+  using fprr = void() &&;
```

Comment:
Please undo this change since it is not related to the patch.

I notice unneeded whitepaces at other places too, please undo these too.